cpp/include/cudf/io/parquet.hpp

```diff
@@ -37,6 +37,9 @@ namespace io {
  * @file
  */
 
+constexpr size_t default_row_group_size_bytes   = 128 * 1024 * 1024;  // 128MB
+constexpr size_type default_row_group_size_rows = 1000000;
```

Comment:
Just to make sure -- we are okay with `size_type`'s maximum value of ~2.1 billion here? Clearly there was a choice to measure bytes with `size_t` and row counts with `size_type = int`.

Reply:
`size_type` is used across cudf for the number of rows. IMO we should be using the same type here. Also, I don't think there's a practical reason to have such huge row groups.